### ISSUE TITLE
EVENT doesn't support the async request with form of application/x-www-form-urlencoded?

### ISSUE THREAD

**Issue by yqleng1987** [dreamhead/moco#238]
Want EVENT to support more properties in the async request like the following:
{
    "request": {
        "uri" : "/event"
    },
    "response": {
        "text": "event"
    },
    "on": {
        "complete": {
            "async" : "true",
            "latency" : 1000,
            "post" : {
                 "url" : "http://another_site",
                 "headers" : {
                    "content-type" : "application/x-www-form-urlencoded"
                 }，
                "forms" : {
                   "first_name" : "Bob",
                    "last_name" : "Lee"
                }
            }
        }
    }
}

**Comment by yqleng1987** [comment]
Since EVENT only support "content" and "json", I tried a workaround as below but it also failed.

```
"on": {
        "complete": {
            "async" : "true",
            "latency" : 1000,
            "post" : {
                 "url" : "http://another_site",
                 "content" : "first_name=Bob&last_name=Lee"
            }
        }
    }
```

In Postman, I can post a request to the callback url "http://another_site" with **form of application/x-www-form-urlencoded** or **form of multipart/form-data** or **putting "first_name" and "last_name" into URL parameters**, and failed with the request json.
It seems that my callback URL doesn't support request json.

**Comment by yqleng1987** [comment]
I need to specify the "content-type" with the value of "application/x-www-form-urlencoded".
But currently EVENT only support "url", "content" and "json". Is that possible to support more fields like "content-type", "forms"?

**Comment by dreamhead** [comment]
You are right. Post in event is just a simple post which needs more support.

**Comment by yayayahei** [comment]
"application/x-www-form-urlencoded" content-type is used frequently, I need this at stub server, wish the owner or anybody can support this. I will try in spare time, but not sure I can do that.![image](https://user-images.githubusercontent.com/16485099/57751752-4c546d80-7719-11e9-9ffc-e5e7289f3067.png)
